tests/load/test_load_case.py

```diff
@@ -19,3 +19,21 @@ def test_load_case(case_obj, panel_database):
 
     for panel in loaded_case['panels']:
         assert panel['display_name']
+
+
+def test_load_case_rank_model_version(case_obj, panel_database):
```

Comment:
Could add an assertment in the beginning that `assert case_obj['rank_model_version'] is not None`or something?